source/spgfn.rst

```diff
@@ -0,0 +1,190 @@
+.. _dipro:
+
+-------------------
+ Spin-polarization
+-------------------
+
+This guide aims to give a general overview of spin-polarized (sp)GFNn-xTB calculations within the ``xtb`` program package.
+
+.. note::
+   This feature is available in ``xtb`` in version 6.6.1 and upward via the tblite.
+
+
+.. contents::
+
+General description
+===================
+
+GFN1-xTB and GFN2-xTB both treat open-shell systems in an restricted open-shell fashion since their energy expressions are spin-independent. 
+Therefore, higher electron spin multiplicities will always be higher in energy than the low-spin multiplicities. 
+However, the methods are occupation dependent and the geometries of high-spin states can still be optimized.
+
+spGFNn-xTB
+===========
+To remedy this issue the spin-polarized GFN1 and GFN2-xTB methods (spGFNn-xTB) have been developed and are available in ``xtb`` via the tblite library with the call: 
+
+.. code:: sh
+   
+   > xtb <geometry_file> --spinpol --tblite
+
+   
+.. note::
+   The spin polarization will only affect the energies of open-shell configurations and not of singlets
+
+Example
+===================
+
+As example we will calculate the spin splittings of a mixed-valence **FeCo** complex with spGFN2-xTB to find the electronic ground state:
+
+.. tabbed:: FeCo
+
+   .. figure:: ../figures/spgfn.png
```

Comment:
As the `sphinx-panels` produces dependency conflicts with newer versions of `sphinx`, it is substituted by the `sphinx-design` extension.
The all `tabbed` directives should be replaced as:
``` rst
.. tab-set::
  .. tab-item:: tab_name
    ...
```